tutorials/text/text_intro.py

```diff
@@ -93,3 +96,325 @@
+# put the xlabel to the far left of the axis.  Note, that the y-coordinate of
+# this position has no effect - to adjust the y-position we need to use
+# the *labelpad* kwarg.
+
+fig, ax = plt.subplots(figsize=(5, 3))
+fig.subplots_adjust(bottom=0.15, left=0.2)
+ax.plot(x1, y1)
+ax.set_xlabel('time [s]', position=(0., 1e6),
+                      horizontalalignment='left')
+ax.set_ylabel('Damped oscillation [V]')
+
+plt.show()
+
+##############################################################################
+# All the labelling in this tutorial can be changed by manipulating the
+# `font_manager.FontProperties` method, or by named kwargs to
+# `Axes.set_xlabel()`
+
+from matplotlib.font_manager import FontProperties
+
+font = FontProperties()
+font.set_family('serif')
+font.set_name('Times New Roman')
+font.set_style('italic')
+
+fig, ax = plt.subplots(figsize=(5, 3))
+fig.subplots_adjust(bottom=0.15, left=0.2)
+ax.plot(x1, y1)
+ax.set_xlabel('time [s]', fontsize='large', fontweight='bold')
+ax.set_ylabel('Damped oscillation [V]', fontproperties=font)
+
+plt.show()
+
+##############################################################################
+# Finally, we can use native TeX rendering in all text objects and have
+# multiple lines:
+
+fig, ax = plt.subplots(figsize=(5, 3))
+fig.subplots_adjust(bottom=0.2, left=0.2)
+ax.plot(x1, np.cumsum(y1**2))
+ax.set_xlabel('time [s] \n This was a long experiment')
+ax.set_ylabel('$\int\ Y^2\ dt\ \ [V^2 s]$')
+plt.show()
+
+
+##############################################################################
+# Titles
+# ======
+#
+# Subplot titles are set in much the same way as labels, but there is
+# the *loc* keyword argumentsthat can change the position and justification
+# from the default value of ``loc=center``.
+
+fig, axs = plt.subplots(3, 1, figsize=(5, 6), tight_layout=True)
+locs = ['center', 'left', 'right']
+for ax, loc in zip(axs, locs):
+    ax.plot(x1, y1)
+    ax.set_title('Title with loc at '+loc, loc=loc)
+plt.show()
+
+##############################################################################
+# Vertical spacing for titles is controlled via ``rcParams[axes.titlepad]``,
+# which defaults to 5 points.  Setting to a different value moves tthe title.
+
+matplotlib.rcParams['axes.titlepad'] = 30.
+fig, ax = plt.subplots(figsize=(5, 3))
+fig.subplots_adjust(top=0.8)
+ax.plot(x1, y1)
+ax.set_title('Vertically offset title')
+plt.show()
+
+
+##############################################################################
+# Ticks and ticklabels
+# ====================
+#
+# Placing ticks and ticklabels is a very tricky aspect of making a figure.
+# Matplotlib does the best it can automatically, but it also offers a very
+# flexible framework for determining the choices for tick locations, and
+# how they are labelled.
+#
+# Terminology
+# ~~~~~~~~~~~
+#
+# *Axes*  (typically) have an `axis.xaxis` and `axis.yaxis` object that
```

Comment:
I think the normal convention is to call a single Axes object "ax" instead of "axis" in order to avoid confusion with Axis.  So "ax.xaxis" and "ax.yaxis".

Reply:
Yes, that was a typo (darn auto-correct).  I changed to `Axes.xaxis` to actually point to the objects.  I think that works.